fetch.bs

```diff
@@ -1517,6 +1517,7 @@ the empty string,
 "<code>style</code>",
 "<code>track</code>",
 "<code>video</code>",
+"<code>webidentity</code>",
```

Comment:
Shouldn't this have been added to the dictionary too, or how is https://fetch.spec.whatwg.org/#dom-request-destination supposed to work?

Reply:
It depends a bit on whether it's exposed to service workers, but if it is then yes. I filed https://github.com/whatwg/fetch/pull/1500 on that.

Reply:
Thanks for pointing that out. It is not exposed to service workers but I'll let Anne handle the question of whether it should be added to the enum anyway, since they have a patch already.